dotcom-rendering/src/web/components/Standfirst.tsx

```diff
@@ -146,6 +146,11 @@ const standfirstStyles = (format: ArticleFormat, palette: Palette) => {
 						margin-bottom: ${space[3]}px;
 						max-width: 540px;
 						color: ${palette.text.standfirst};
+
+						a:hover {
+							border-bottom: solid 0.0625rem
+								${palette.hover.liveBlogStandfirst};
```

Comment:
Can we go with `1px` to be consistent. There are a handful of cases where rems are used in DCR but for the most part it's using pixels

Reply:
This was just to maintain parity with frontend but I'll change it to 1px